starters/knative/knative-junit-starter/pom.xml

```diff
@@ -14,7 +14,7 @@
   <dependencies>
```

Comment:
Should we keep maintaining this `knative-junit-starter`? I don't think it makes sense any longer as users have know the `knative-junit` extension.